designer/server/src/routes/account/sign-out.js

```diff
@@ -3,16 +3,20 @@ import { URL } from 'node:url'
 import Boom from '@hapi/boom'
 
 import { dropUserSession } from '~/src/common/helpers/auth/drop-user-session.js'
+import { hasUser } from '~/src/common/helpers/auth/get-user-session.js'
 import config from '~/src/config.js'
 
+const redirectUrl = new URL(`/account/signed-out`, config.appBaseUrl)
+
 export default /** @satisfies {ServerRoute} */ ({
   method: 'GET',
   path: '/auth/sign-out',
   async handler(request, h) {
-    await dropUserSession(request)
+    const { credentials } = request.auth
 
-    // Skip OpenID Connect (OIDC) in tests
-    if (config.isTest) {
+    // Skip OpenID Connect (OIDC) when not authenticated
+    if (!hasUser(credentials) || config.isTest) {
+      await dropUserSession(request)
```

Comment:
We're appending `id_token_hint` to the redirect so users should be signed in

Similarly we might want to try `logout_hint` via `credentials.user.email` in future should the "Pick an account" prompt continue to appear